packages/web3-utils/tests/src/UtilsTest.js

```diff
@@ -369,9 +370,20 @@ describe('UtilsTest', () => {
         expect(hexToNumberString('0x1f0fe294a36')).toEqual('2134567897654');
 
         // allow compatiblity
-        expect(hexToNumberString(100000)).toEqual('100000');
+        expect(hexToNumberString(100000)).toEqual('1048576');
```

Comment:
I think this should throw an error. It should only accept strings.

Reply:
Yes right. I will change the test.